app/manifest.json

```diff
@@ -27,13 +27,18 @@
                 },
                 {
                     "name": "UseTLS",
-                    "default": "yes",
+                    "default": "no",
                     "type": "enum:no|No, yes|Yes"
                 },
                 {
-                    "name": "IPCSocket",
+                    "name": "TCPSocket",
```

Comment:
For docker-acap lets have TCP to default "yes" - for docker-compose-acap it should be default "no"